.github/workflows/test.yml

```diff
@@ -18,9 +18,10 @@ jobs:
         - 10
         - 12
         - 14
+        - 16
     services:
       mongodb:
-        image: mongo:3.6
+        image: mongo:4.4
```

Comment:
Just so I understand, this is not a breaking change, because sharedb is "db agnostic", correct?

Reply:
Yeah, this is specifically just updating the version of MongoDB we use in the [build step](https://github.com/share/sharedb/blob/a837b0090ef4731ca4605d067a40a61e6ef0410c/.github/workflows/test.yml#L45) where we sense-check against `sharedb-mongo` (which should support all current MongoDB versions anyway)